pkgs/tools/misc/chafa/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, makeWrapper, pkgconfig, gtk-doc, autoconf, automake, libtool, glib, imagemagick, which }:
+
+stdenv.mkDerivation rec{
+  version = "1.0.1";
+  name = "chafa-${version}";
+
+  src = fetchFromGitHub {
+    owner = "hpjansson";
+    repo = "chafa";
+    rev = "${version}";
+    sha256 = "1i1cnzmb12pxldc7y4q1xdmybv9xkhzrjyhdvmk3qsn02p859q04";
+  };
+
+  buildInputs = [ autoconf automake libtool pkgconfig makeWrapper gtk-doc glib imagemagick which ];
```

Comment:
Everything except `glib` and `imagemagick` should be in `nativeBuildInputs`, because it's only used during the build. And `makeWrapper` doesn't seem to be used.